marketplacetests/marketplace/regions/search_results.py

```diff
@@ -25,6 +23,12 @@ def search_results(self):
         search_results = self.marionette.find_elements(*self._search_result_locator)
         return [Result(self.marionette, result) for result in search_results]
 
+    def install_searched_app(self, search_term):
+        # Select the application link in the list
```

Comment:
I don't think these comments are necessary. The second one doesn't make much sense to me in the context of the code. The method itself is self-explanatory.